### PR TITLE
add date picker to input component

### PR DESCRIPTION
**Pull request by JuanVimon3**

### CODE REVIEW

File: src/containers/Home/styles.module.css [hunk 1]
```diff
@@ -15,3 +16,12 @@
 .dropdownContainer{
   margin: auto 30px
 }
+
+.sourceFromDateContainer{
+  margin: auto 30px;
+  margin-left: 135px;
```
Comment (gualopezb) [inline src/containers/Home/styles.module.css:22]: We may need to change the approach to style this container to get rid of some issues. Let's discuss this on a call.

**Review by gualopezb — CHANGES_REQUESTED** [review]

File: src/components/Input/index.jsx [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
       {labelText}
     </label>
     <input
       id="input"
       value={value}
-      type="text"
+      type={type}
       onChange={onChangeValue}
-      className={styles.input}
+      className={`${styles.input} ${value === '' ? styles.placeholderActive : ''}`}
```
[the remark this answers is not in the document]
Comment (JuanVimon3) [reply src/components/Input/index.jsx:14]: image Hello, Gustavo. I don't undertstand this comment. If I delete this line wich class should I use to put the placeholder prop in gray?

File: src/components/Input/index.jsx [hunk 3]
```diff
@@ -1,32 +1,39 @@
...
       placeholder={placeholder}
+      onFocus={onFocus}
+      onBlur={onBlur}
     />
   </div>
 );
 
 Input.propTypes = {
   value: PropTypes.string.isRequired,
+  type: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (JuanVimon3) [reply src/components/Input/index.jsx:24]: Did we delete the labelText prop? 

image

File: src/components/Input/styles.module.css [hunk 4]
```diff
@@ -17,3 +17,6 @@
   margin-bottom: 5px;
   font-size: 14px;
 }
+.placeholderActive {
```
[the remark this answers is not in the document]
Comment (JuanVimon3) [reply src/components/Input/styles.module.css:20]: Oh, got it!

File: src/containers/Home/styles.module.css (hunk 1 above)
Comment (JuanVimon3) [reply src/containers/Home/styles.module.css:22]: Ok, I'll be waiting!

File: src/components/Input/index.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (gualopezb) [reply src/components/Input/index.jsx]: @JuanVimon3 you don't need to do anything special here, the placeholder for the input is just gray by default. If you recall, we had to do this trick for the dropdown implementation, but in this particular case of the input is not needed.

File: src/components/Input/index.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (gualopezb) [reply src/components/Input/index.jsx:25]: @JuanVimon3 no, it's still there. If you take a closer look at the file, it's still a prop and it's indeed used in the line 8.

[the remark this answers is not in the document]
Comment (gualopezb) [reply src/components/Input/index.jsx:25]: BTW, my suggestion is just about listing out the properties of the `Input.propTypes` object in the same order as they appear in the function definition.

File: src/containers/Home/index.jsx [hunk 5]
```diff
@@ -2,13 +2,16 @@ import { useState } from 'react';
 
 import Input from '../../components/Input';
```
Comment (gualopezb) [inline src/containers/Home/index.jsx:4]: Please remove this empty line. Custom modules should be imported altogether in the same block.
Just to make sure we're on the page here, this is the convention we've been trying to follow:

```
imports of external libraries (e.g. react, prop-types, etc.)
empty line
imports of custom modules (e.g. own react componentes)
empty line
imports of styles
```

For instance: 
```
import { useState } from 'react';

import Input from '../../components/Input';
import Dropdown from '../../components/Dropdown';

import styles from './styles.module.css';
```

Comment (gualopezb) [reply src/containers/Home/index.jsx:4]: LMK if this is not clear and we can discuss it.

Comment (JuanVimon3) [reply src/containers/Home/index.jsx]: Got it

**Review by gualopezb — APPROVED** [review]

**Pull request merged**